### PR TITLE
docs: add support for supply chain security

### PR DESCRIPTION
**Pull request by osamamagdy**
Documentation for steps in https://github.com/jenkins-x/jx/issues/8348

### CODE REVIEW

File: content/en/community/maintainer_guide/supply-chain-security.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+title: Add Supply Chain Security
+linktitle: Add Supply Chain Security
+description: Internal infrastructure used for Jenkins X
```
Comment (ankitm123) [inline content/en/community/maintainer_guide/supply-chain-security.md:4]: wrong description

File: content/en/community/maintainer_guide/supply-chain-security.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+weight: 400
+type: docs
+no_list: true
+---
+## Introduction
+
+Jenkins X started to support suplly chain security through different steps.
```
Comment (ankitm123) [inline content/en/community/maintainer_guide/supply-chain-security.md:11]: remember to run spell check, typo: suplly

File: content/en/community/maintainer_guide/supply-chain-security.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    syft ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:latest --scope all-layers -o spdx-json > sbom.json
+    echo $GITHUB_TOKEN | oras push -u $GIT_USERNAME --password-stdin ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:$VERSION-sbom sbom.json
+    echo $GITHUB_TOKEN | oras push -u $GIT_USERNAME --password-stdin ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:latest-sbom sbom.json
+    ```
+
+
+- For repositories that use JX itself to release ([jx-pipeline](https://github.com/jenkins-x-plugins/jx-pipeline) for example):
+
+  - First, include those two steps to use kaniko to build the image and crane to push it (image.tar is required for syft to generate SBOM)
```
Comment (ankitm123) [inline content/en/community/maintainer_guide/supply-chain-security.md:80]: may be write it differently. Replace the build-push-image step with these two steps.

File: content/en/community/maintainer_guide/supply-chain-security.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
+---
+title: Add Supply Chain Security
```
Comment (ankitm123) [inline content/en/community/maintainer_guide/supply-chain-security.md:2]: may be change this to `Add sbom generation step to release pipelines`

**Review by ankitm123 — COMMENTED** [review]
minor comments

File: content/en/community/maintainer_guide/supply-chain-security.md (hunk 1 above)
Comment (osamamagdy) [reply content/en/community/maintainer_guide/supply-chain-security.md:4]: Done

File: content/en/community/maintainer_guide/supply-chain-security.md (hunk 4 above)
Comment (osamamagdy) [reply content/en/community/maintainer_guide/supply-chain-security.md:2]: Done

File: content/en/community/maintainer_guide/supply-chain-security.md (hunk 3 above)
Comment (osamamagdy) [reply content/en/community/maintainer_guide/supply-chain-security.md:80]: Done

**Review by ankitm123 — APPROVED** [review]

**Pull request merged**